### PR TITLE
Fix problem with regme saying user does not exists when it's not the case

### PR DESCRIPTION
**Pull request by tokongs**
## What kind of a pull request is this?

- QA / Code Review

## Code Checklist

- [x] The code follows dotkom code style 
- [x] The code passes the defined tests
- [ ] I have added tests for the code I added
- [ ] I have provided documentation for the code I added
- [x] The code is ready to be merged

## (Possible) Breaking Changes

- [x] The changes are backwards compatible

- [ ] I have updated the build configuration
- [ ] These changes requires changes to configuration in production 
    - [ ] I have applied the required changes in production
    - [ ] I cannot apply the required changes in production before this is deployed.

## Description of changes
Added a check to see if user exists in database when the user is not found as an attendee at event registration. 
FIxes issue #2052

### CODE REVIEW

**Comment by Aborysa** [comment]
Looks good, just remember to add a case for this in the regme app before merging this 👍

File: apps/events/views.py [hunk 1]
```diff
@@ -504,20 +504,32 @@ def post(self, request, format=None):
...
+                return Response({'message': (user[0].get_full_name() +
+                                '            er ikke meldt på dette arangementet.'),
+                                'attend_status': 55},
+                                status=status.HTTP_403_FORBIDDEN)
 
         # All is clear, attendee is attended
-        return Response({'message': (attendee.user.get_full_name() + ' er registrert som deltaker. Velkommen!'),
-                         'attend_status': 10, 'attendee': attendee.id}, status=status.HTTP_200_OK)
+        return Response({'message': (user.get_full_name() + ' er registrert som deltaker. Velkommen!'),
+                         'attend_status': 10, 'attendee': 2}, status=status.HTTP_200_OK)
```
Comment (sklirg) [inline apps/events/views.py]: Why `2`?

Comment (tokongs) [reply apps/events/views.py]: That was me testing something and forgetting to remove it. Will make a commit later today.

**Comment by sklirg** [comment]
Furthermore, it looks like some of the test cases breaks with this change.

**Comment by Aborysa** [comment]
Regme isn't worked on currently and probably won't be worked on for a long time.

**Pull request closed without merging**